tests/utils/test_schemapi.py

```diff
@@ -629,13 +638,25 @@ def chart_error_example__four_errors():
     )
 
 
-@pytest.mark.parametrize(
-    ("chart_func", "expected_error_message"),
-    [
-        (
-            chart_error_example__invalid_y_option_value_unknown_x_option,
-            inspect.cleandoc(
-                r"""Multiple errors were found.
+def id_func(val) -> str:
+    """
+    Ensures the generated test-id name uses only `chart_func` and not `expected_error_message`.
+
+    Without this, the name is ``test_chart_validation_errors[chart_func-expected_error_message]``
+    """
+    if isinstance(val, types.FunctionType):
+        return val.__name__
+    else:
+        return ""
+
+
+# NOTE: Avoids all cases appearing in a failure traceback
+# At the time of writing, this is over 300 lines
+chart_funcs_error_message: list[tuple[Callable[..., Any], str]] = [
+    (
+        chart_error_example__invalid_y_option_value_unknown_x_option,
+        inspect.cleandoc(
```

Comment:
Can we remove this `inspect.cleandoc` as well as you have done for the others? It seems to work for me locally.